kvbc/include/merkle_tree_db_adapter.h

```diff
@@ -181,6 +181,8 @@ class DBAdapter : public IDbAdapter {
 
   block::detail::Node getBlockNode(BlockId blockId) const;
 
+  KeysVector staleIndexKeysForBlock(BlockId blockId) const;
+
   KeysVector staleIndexKeysForVersion(const sparse_merkle::Version &version) const;
```

Comment:
Maybe we can rename that to indicate these are stale tree keys. `staleIndexKeysForBlock `, on the other hand, can be named in a way to indicate it relates to non-provable keys.

Reply:
done